src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.ListView.cs

```diff
@@ -383,25 +455,13 @@ private void TasView_ColumnClick(object sender, InputRoll.ColumnClickEventArgs e
 				}
 				else if (columnName != CursorColumnName)
 				{
-					var frame = TasView.AnyRowsSelected ? TasView.FirstSelectedRowIndex : 0;
 					var buttonName = TasView.CurrentCell.Column!.Name;
 
 					if (ControllerType.BoolButtons.Contains(buttonName))
 					{
 						if (ModifierKeys != Keys.Alt)
 						{
-							// nifty taseditor logic

```

Comment:
I remember writing that line but I don't remember what was nifty about taseditor logic. Can you explain in text what the problem was and how you fixed it?

Reply:
Oh I see. Yeah that's a bug then. TASEditor handles this correctly. Tho I still can't tell if the code is good :D